llvm/test/lab2/kostin_artem/loop_plugin_test.ll

```diff
@@ -0,0 +1,88 @@
+; RUN: opt -load-pass-plugin=%llvmshlibdir/loopStartEndPlugin%shlibext -passes=loop-start-end -S %s | FileCheck %s
+
+; void foo(int n, int m) {
```

Comment:
Please add tests for:
- while loop
- do-while loop
- function with switch statement but without loops
- a loop with additional return instruction inside 
- loop with already inserted loop_* calls